src/elements/content-sidebar/Sidebar.js

```diff
@@ -107,11 +107,13 @@ class Sidebar extends React.Component<Props, State> {
     }
 
     componentDidMount() {
-        const { file, api, metadataSidebarProps, docGenSidebarProps }: Props = this.props;
+        const { file, api, metadataSidebarProps, docGenSidebarProps, onOpenChange = noop }: Props = this.props;
         // if docgen feature is enabled, load metadata to check whether file is a docgen template
         if (docGenSidebarProps.enabled) {
             docGenSidebarProps.checkDocGenTemplate(api, file, metadataSidebarProps.isFeatureEnabled);
         }
+        const isOpen = this.isOpen();
+        onOpenChange(isOpen, true);
```

Comment:
nit: can we pass `this.isOpen` directly to the function without creating a new const?